### DISCUSSION TITLE
Render parsed Json file as HTML

### DISCUSSION THREAD

**Discussion by rhnprabhune** [DS4SD/deepsearch-toolkit#130] [Q&A]
Hi. I have used the deep search toolkit (python) to parse a pdf document to get a json file.  However when I use the web tool I see that json file is rendered to UI as an HTML.

![image](https://github.com/DS4SD/deepsearch-toolkit/assets/51504486/fb4ca070-a8d2-4f04-a9b4-a442d8aad008)

I was wondering if there is way I can render the parsed json file (from toolkit) as an HTML myself ?

**Comment by dolfim-ibm** [comment]
@rhnprabhune Thanks for your question. What you are proposing is indeed very close to some updates we are planning, which will allow to operate with the output document in a simple way. For example, iterate through figures, convert to HTML, etc.

We actually have some code snippet around which we want to centralize in the toolkit for everybody.
Will keep you posted.

**Reply by rhnprabhune** [reply]
Hi @dolfim-ibm Thank you for your response. Waiting for updates on this :) 
Do keep me posted
